layers/+spacemacs/spacemacs-evil/packages.el

```diff
@@ -155,6 +156,12 @@
       (define-key evil-visual-state-map evil-exchange-cancel-key
                   'evil-exchange-cancel))))
 
+(defun spacemacs-evil/init-evil-fringe-mark ()
+  (use-package evil-fringe-mark
+    :config
+    (setq-default evil-fringe-mark-show-special t)
+    (global-evil-fringe-mark-mode)))
```

Comment:
If we add this package, it should be deferred, and the mode should not be enabled by default. It would make sense to add a toggle as for `evil-visual-mark-mode`, <kbd>SPC t '</kbd> seems logical as a key binding.